blog.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: blog
+title: Blog
+slug: blog
+---
+
+<h1 class="h2 text-center mb-4">The Sustain Blog</h1>
+
+<section class="pb-5">
```

Comment:
Maybe just replace it with a `<div>` - https://www.impressivewebs.com/html5-section/

Reply:
@coderberry